src/js/angular/clustermanagement/controllers.js

```diff
@@ -130,19 +131,31 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $modal,
             .then((response) => {
                 $scope.clusterConfiguration = response.data;
                 if (!$scope.currentNode) {
-                    $scope.getCurrentNodeStatus();
+                    return $scope.getCurrentNodeStatus();
                 }
             })
             .catch(() => {
                 $scope.clusterConfiguration = null;
             });
     };
 
+    function isLeaderChanged(currentLeader, newLeader) {
+        if (!newLeader) {
+            return true;
```

Comment:
shouldn't this be `false`?

Reply:
If there was an elected leader and he died, I consider it a leader change although there is no new leader at that time.

Reply:
Well. Maybe add this comment in the code, because it's somewhat confusing

Reply:
added comment